lib/src/oauth2_flows/metadata_host_io.dart

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) 2019, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+library googleapis_auth.metadata_server_flow;
+
+import 'dart:io';
+
+const _GCE_METADATA_HOST_ENV_VAR = "GCE_METADATA_HOST";
```

Comment:
If this convention is documented somewhere it would be lovely to reference said docs.

Reply:
I was just going off of Nate's suggestion. Perhaps he can chime in here with a link?

Reply:
I don't think we have one yet. @kevmoo has started working on one.

Reply:
Wait, sorry :) I was asking if the environment variable name `GCE_METADATA_HOST` is documented somewhere.

------
Side note, the only documentation I could find for using the metadata service is:
https://cloud.google.com/compute/docs/storing-retrieving-metadata

Which currently says to use `metadata.google.internal`, rather than `metadata`.
Granted, we're already using `metadata`, so presumably that also works :)

Reply:
Spoke off of Github. Unfortunately I also don't know / couldn't find any documentation for this. Jonas is okay to leave this as is (for lack of a proper reference).